internal/database/drivers/mongodb/mongodb.go

```diff
@@ -167,7 +167,7 @@ func (m *Database) GetEvents(ctx context.Context, request requests.MultipleEvent
 	}
 
 	m.logger.Debugf("fetching events from %d collections", len(collections))
-	var allEvents []drivers.EiffelEvent
+	allEvents := make([]drivers.EiffelEvent, 0)
```

Comment:
An optional optimization would be to make sure that the slice has room for all the documents we may want to return to avoid reallocations later on.

    allEvents := make([]drivers.EiffelEvent, 0, request.PageSize)